src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -705,16 +705,16 @@ export default {
     },
 
     /** Returns True if the specified business has any pending tasks, else False. */
+    // FUTURE move this to Legal API mixin
     async hasTasks (businessId) {
       let hasPendingItems = false
       if (this.filingId === 0) {
         const url = `businesses/${businessId}/tasks`
         await axios.get(url)
           .then(response => {
-            if (response && response.data && response.data.tasks) {
+            if (response?.data?.tasks) {
               response.data.tasks.forEach((task) => {
```

Comment:
This could probably be improved with a `find()` (or `some()`) so it doesn't iterate over all tasks.